spacy/tokens/span_group.pyx

```diff
@@ -170,6 +170,16 @@ cdef class SpanGroup:
             raise ValueError(Errors.E855.format(obj="span"))
         self.push_back(span.c)
 
+    def __iter__(self):
+        """
+        Iterate over the spans in this SpanGroup.
+        YIELDS (Span): A span in this SpanGroup.
+
+        DOCS: https://spacy.io/api/spangroup#iter
```

Comment:
And I think the last step is to add this to the API docs.

Reply:
Docs are updated in https://github.com/explosion/spaCy/pull/12009/commits/19657b392a5fab8392be1e518fefacb442fd78a3.